test/jdk/javax/swing/JButton/HtmlButtonImageTest/HtmlButtonImageTest.java

```diff
@@ -115,41 +113,41 @@ private static void createAndShowGUI()
         frame.setVisible(true);
     }
 
-    private static void setupCenterCoord() {
+    private static void setupCenterCoord() throws InterruptedException, InvocationTargetException {
         // adjust coordinates to be the center of the button
-        point = button.getLocationOnScreen();
+        SwingUtilities.invokeAndWait(() -> {
+            point = button.getLocationOnScreen();
+        });
         point.x += BUTTON_WIDTH / 2;
         point.y += BUTTON_HEIGHT / 2;
     }
 
     private static boolean checkRedness(Color c) {
         // checks for redness since anti-aliasing causes edges to be not exactly 255,0,0 rgb values
-        if(c.getRed() > 250 && c.getBlue() < 10 && c.getGreen() < 10) {
+        if (c.getRed() > 250 && c.getBlue() < 10 && c.getGreen() < 10) {
             return true;
         }
         return false;
     }
 
-    private static void testImageCentering(Color left, Color right, Color top, Color bottom) {
+    private static void testImageCentering(Color... colors) {
         // check if all colors at each edge of square are red
-        if(!checkRedness(left) || !checkRedness(right) || !checkRedness(top) || !checkRedness(bottom)) {
-            throw new RuntimeException("HTML image not centered in button");
-        }
-        else {
-            System.out.println("-- Passed");
+        for (Color c : colors) {
+            if (!checkRedness(c)) {
+                throw new RuntimeException("HTML image not centered in button");
+            }
+            else {
+                System.out.println("-- Passed");
+            }
```

Comment:
You don't need the else block. Now this message is printed four times. Just print the message after the for-loop.